### PR TITLE
feat: Add gas_used to kakarot.execute_at_address returned values

### PR DESCRIPTION
**Pull request by mradolux**
Time spent on this PR: 0.5 day

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

execute_at_address doesn't return gas_used

Resolves https://github.com/sayajin-labs/kakarot/issues/428

execute_at_address does return gas_used

- added gas used to return values of execute_at_address function

### CODE REVIEW

**Review by ClementWalter — CHANGES_REQUESTED** [review]
some tests are skipped because of this missing returned value, can you update them? (search for gas_used or gas_usage)

**Comment by mradolux** [comment]
The failing test is not caused by the gas_used variable, it is caused by the createPair function. We can't test the gas_used variable before fixing the "StarknetErrorCode.OUT_OF_RESOURCES: 42" caused by the createPair

**Comment by ClementWalter** [comment]
you should be able to pass tests updating the config given to `Starknet.empty(...)`

- in file `1_starknet.py`
- import `from starkware.starknet.definitions.general_config import StarknetGeneralConfig`
- right before `starknet = await Starknet.empty()` in the fixture
   - create a config `config = StarknetGeneralConfig(invoke_tx_max_n_steps=2**24, validate_max_n_steps=2**24)`
   - use this config `starknet = await Starknet.empty(config)`

**Comment by mradolux** [comment]
The gas_used of the tx doesn't correspond to the value written in the test, should we update it or should the gas_used of the tx be equal to the value written in the test ?

**Review by ClementWalter — APPROVED** [review]

File: tests/integration/solidity_contracts/UniswapV2/test_uniswap_v2_factory.py [hunk 1]
```diff
@@ -71,13 +71,11 @@ async def test_should_create_pair_only_once(
             assert await pair.token0() == tokens[0]
             assert await pair.token1() == tokens[1]
 
-        @pytest.mark.skip("gas_usage is not yet returned by kakarot")
         async def test_should_use_correct_gas(self, factory, owner):
-            # TODO: see https://github.com/sayajin-labs/kakarot/issues/428
             tx = await factory.createPair(
```
[the remark this answers is not in the document]
Comment (mradolux) [reply tests/integration/solidity_contracts/UniswapV2/test_uniswap_v2_factory.py:76]: How can I activate the CI on my forked repo ?

[the remark this answers is not in the document]
Comment (jobez) [reply tests/integration/solidity_contracts/UniswapV2/test_uniswap_v2_factory.py:76]: image

I think its just clicking a button prompt that shows up when you click the actions tab in your forked repo.

[the remark this answers is not in the document]
Comment (mradolux) [reply tests/integration/solidity_contracts/UniswapV2/test_uniswap_v2_factory.py:76]: I think it's good now

**Pull request merged**